No rule found for src/ocamlorg_toplevel/bin/toplevel_base.bc when trying to build following readme

I checked out the repo and followed the setup instructions:

```
make switch
```

But running `make build` gives:

```
x  make build
opam exec -- dune build --root .
File "src/ocamlorg_toplevel/bin/dune", line 32, characters 0-117:
32 | (rule
33 |  (targets export_base.txt)
34 |  (deps toplevel_base.bc)
35 |  (action
36 |   (run jsoo_listunits -o %{targets} stdlib base)))
Error: No rule found for src/ocamlorg_toplevel/bin/toplevel_base.bc
File "src/ocamlorg_toplevel/bin/dune", line 72, characters 0-116:
72 | (rule
73 |  (targets export_stdlib.txt)
74 |  (deps toplevel_stdlib.bc)
75 |  (action
76 |   (run jsoo_listunits -o %{targets} stdlib)))
Error: No rule found for src/ocamlorg_toplevel/bin/toplevel_stdlib.bc
make: *** [Makefile:29: build] Error 1   
```

This is on `main`.

--- a/src/ocamlorg_toplevel/bin/dune
+++ b/src/ocamlorg_toplevel/bin/dune
@@ -27,7 +27,7 @@
  (name toplevel_base)
  (libraries ocamlorg_toplevel base)
  (modules toplevel_base)
- (modes js))
+ (modes byte js))
 
 (rule
  (targets export_base.txt)
@@ -65,7 +65,7 @@
  (name toplevel_stdlib)
  (libraries ocamlorg_toplevel stdlib)
  (modules toplevel_stdlib)
- (modes js))
+ (modes byte js))
 
 ; Stdlib worker
 

